breakpad-symbols/src/sym_file/walker.rs

```diff
@@ -737,7 +737,7 @@ enum CfiReg<'a> {
     Other(&'a str),
 }
 
-fn eval_win_expr(expr: &str, info: &StackInfoWin, walker: &mut dyn FrameWalker) -> Option<()> {
```

Comment:
I'll need to make sure this wasn't made public when it shouldn't have been, I needed it for the fuzzer to call.

Reply:
it was

i put it behind the same cfg(feature = fuzz) as the re-exports were.